startleft/api/api_config.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+
+class ApiConfig:
+    iriusrisk_server: str
+
+    @classmethod
+    def set_iriusrisk_server(cls, server: str):
+        cls.iriusrisk_server = server
+
+    @classmethod
+    def get_iriusrisk_server(cls):
+        if not hasattr(cls, 'iriusrisk_server'):
```

Comment:
This verification is not required, as the set_iriusrisk_server is being always called with either the given parameter or the environment variable.